src/metabase/driver/bigquery.clj

```diff
@@ -411,12 +406,16 @@
           :describe-table        (u/drop-first-arg describe-table)
           :details-fields        (constantly [{:name         "project-id"
                                                :display-name "Project ID"
-                                               :placeholder  "praxis-beacon-120871"
+                                               :placeholder  "bigquery-public-data"
                                                :required     true}
                                               {:name         "dataset-id"
                                                :display-name "Dataset ID"
-                                               :placeholder  "toucanSightings"
+                                               :placeholder  "baseball"
```

Comment:
We could always add a `(optional)` or something like that in the names or in some other way